roles/matrix-bot-matrix-registration-bot/templates/env.j2

```diff
@@ -0,0 +1,8 @@
+BOT_SERVER={{ matrix_bot_matrix_registration_bot_bot_server }}
+BOT_USERNAME={{ matrix_bot_matrix_registration_bot_matrix_user_id_localpart }}
+BOT_ACCESS_TOKEN={{ matrix_bot_matrix_registration_bot_api_token }}
```

Comment:
Why is it called `bot_api_token` when it influences the `bot_access_token` variable? It should probably be called `matrix_bot_matrix_registration_bot_bot_access_token`. That `bot_bot_` prefix might look weird, but it's because the role name is `matrix-registration-bot` (which gives us the `matrix_bot_matrix_registration_bot_` part), and then we have the actual property name (`bot_access_token`).

Why use environment variables? I thought you're moving all this to `config.yml` only? Or you abandoned that idea?

Reply:
Sorry I forgot to delete that file it is no longer needed as I indeed changed to a config file